foundation/plugins/org.jboss.tools.foundation.ui/src/org/jboss/tools/foundation/ui/credentials/internal/CredentialMessages.properties

```diff
@@ -18,3 +18,14 @@ SelectDomain=Please select a domain.
 UsernameCannotBeBlank=Username cannot be blank.
 PasswordCannotBeBlank=Password cannot be blank.
 UsernameAlreadyExists=Username {0} already exists for domain {1}.
+
+# For use in the credential prompter
+OK=OK
+Cancel=Cancel
+CredentialPrompterTitle=Password Required
```

Comment:
The Associated Press has style guidelines for titles. I'm not sure if this applies to eclipse or not, but the Associated Press tends to capitalize all words that are not prepositions. 

https://www.sandiego.edu/documents/its/web/brief_ap_style.pdf

Capitalizing 
Avoid random capitalization.  Do not capitalize for emphasis. Do not use exclamation 
points. 
1. In a headline.
 Capitalize all words in a title or  headline except articles (a, an, the) 
and prepositions (of, on, to, at, in). Do not use all caps. (Students Engage in 
Community Outreach)  

I think this can be debated, and I welcome a debate on the topic on jbt-dev ;) Am I just being argumentative?